src/utils/commands/command_abc.py

```diff
@@ -41,6 +41,16 @@ def _rank(self, **kwargs) -> str:
         """
         pass
 
+    @abstractmethod
+    def _completion_check(self, **kwargs) -> int:
+        """
+        Helper function for status & group_status to check for question completion
+        Keyword Args:
+         - question         The question to be checked
+         - leetcode_user_id The leetcode user ID
+        """
+        pass
+
```

Comment:
I'm not convinced that this needs to be within the abstract class for commands. It is more of a helper function for the LeetcodeQuestions class

Reply:
Thanks, good to confirm. I've removed this function.